### ISSUE TITLE
Create a block @declare to declare a variable in template

### ISSUE THREAD

**Issue by mohamed-badaoui** [angular/angular#57617]
### Description

Create a block @declare to declare a variable in template instead of using directive like :
```
<ng-container *ngVar="form.errors as error">
...
</ng-container>
```

### Proposed solution

for example 
@declare(const errors = form.errors)

### Alternatives considered

@declare { const errors = form.errors }

**Comment by JeanMeche** [comment]
This feature has already landed in v18.1 with `@let`. 

https://angular.dev/guide/templates/let-template-variables

**Comment by mohamed-badaoui** [comment]
@JeanMeche thank you very much!
